src/App.tsx

```diff
@@ -32,14 +36,19 @@ function App() {
         <div className="search-section">
           <SearchBox controller={searchBox} />
         </div>
+
         <div className="main-section">
           <div className="facet-section column">
             <Facet controller={facet} title="Source" />
           </div>
           <div className="results-section column">
+            <div className="query-section">
```

Comment:
Thank you for the suggestion, but I think it's best not to add new components. This tutorial app was kept very simple on purpose, to ease the learning experience.